### ISSUE TITLE
Some super awesome weekly issue

### ISSUE THREAD

**Issue by jihadichan** [jihadichan/jp#1]
# Comfy af

Images via copy and paste.

![image](https://user-images.githubusercontent.com/60828589/81851376-a28d0400-9559-11ea-9f50-22afbf2b9ad8.png)

Formatting in Markdown and shit

- something something.
- something something.
- something something.

Direct links: https://9gag.com/

Why not some code:

```java
if(isComfy) {
    getTheFuckInHere();
}
```

**Comment by jihadichan** [comment]
Some reply: haha top kek and shit.

**Comment by jihadichan** [comment]
But what about gifs?
![image](https://user-images.githubusercontent.com/60828589/81852011-a53c2900-955a-11ea-99fb-a639702e4149.png)

**Comment by jihadichan** [comment]
Nope, doesn't work. Get converted to PNG.

**Comment by suicideguy1** [comment]
![1556655755634](https://user-images.githubusercontent.com/65342017/81923437-f7725e00-95dd-11ea-9197-d817fa600fe2.jpg)
testing

**Comment by jihadichan** [comment]
@suicideguy1 cool, eh?

**Comment by suicideguy1** [comment]
> @jihadichan
Testing again.

**Comment by jihadichan** [comment]
@suicideguy1 this shit even has live updates. didn't know that.

**Comment by suicideguy1** [comment]
@jihadichan Hm, i'm not receiving any pings. Maybe its because this is your own thread?

**Comment by jihadichan** [comment]
@suicideguy1 same problem here. Weird. Let me check.

**Comment by suicideguy1** [comment]
@jihadichan You can tweak with the settings.

**Comment by jihadichan** [comment]
Fuck. Notifications come via eMail ...

**Comment by suicideguy1** [comment]
@jihadichan I received your latest notification via email as well.

**Comment by jihadichan** [comment]
mmmh... do you know Slack? It's also actually made for communication... and is not an issue tracker...

**Comment by suicideguy1** [comment]
https://slack.com/intl/en-se/ this one? We could try it.

**Comment by jihadichan** [comment]
yep, it's for team communication. Like Skype, but actually good. And perfect for sharing memes.

**Comment by suicideguy1** [comment]
Ok, im in. my User is the same as here "suicideguy1"

**Comment by suicideguy1** [comment]
seems like my slack address is team-shitposting.slack.com

**Comment by jihadichan** [comment]
Cool, could you invite jihadichan@web.de ?
